python/ray/tests/test_multi_node_2.py

```diff
@@ -265,6 +265,89 @@ def f():
     assert ray_client.get(f.remote()) == "hello client"
 
 
+def test_detached_actor_autoscaling(ray_start_cluster_head):
```

Comment:
Did you check these tests fail without the fixes?

Reply:
yup